src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import { SideMenu, Platform, Resources, Solutions } from ".";
+import Button from "../Reusable/Button";
```

Comment:
@Rauniyar-shovit please create an index.ts file inside .src/Reusable/inde.ts

And export Button 

export * from "./Button"

Component name "Button" can be very confusing, since we can have multiple types of button component throughout the project. Please call it Primart, Secondary or Tertiary Button